src/lib.rs

```diff
@@ -210,6 +210,17 @@ impl Requirement {
         self.extras.clone()
     }
 
+    /// The marker expression such as  `python_version > "3.8"` in
+    /// `requests [security,tests] >= 2.8.1, == 2.8.* ; python_version > "3.8"`
+    #[getter]
+    pub fn marker(&self) -> Option<String> {
+        if self.marker.is_some() {
+            Some(self.marker.as_ref().unwrap().to_string())
+        } else {
+            None
+        }
```

Comment:
You can use [map](https://doc.rust-lang.org/nightly/std/option/enum.Option.html#method.map) here to make this a single line call

Reply:
oh, rust is nice :-)

Reply:
added a minimal test